spec/console/console_io_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "console/console_io"
+
+describe ConsoleIO do
+  let(:error_message) { "Sorry I didn't recognise that" }
+
+  it "prints messages to the screen" do
+    output = StringIO.new
+    console_io = ConsoleIO.new(output: output)
+    console_io.puts("hello")
+
+    expect(output.string).to eq("hello\n")
+  end
+
+  it "reads an integer in a given range" do
+    input = StringIO.new("3")
+    output = StringIO.new
+    console_io = ConsoleIO.new(input: input, output: output)
+
+    result = console_io.read_int_in_range(1, 9)
+    expect(result).to eq(3)
+  end
+
+  it """prints an error message for non integer input
```

Comment:
any reason for the double double quotes? 😂

Reply:
lol it was for a multiline string, but maybe I don't need them (also probably a sign the test / method is doing too much)